assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -270,17 +273,25 @@ def run(args):
         log_time_and_message("Reading data in spark and converting to pandas")
         baseline_df = try_read_mltable_in_spark_with_error(args.baseline_data, "baseline_data")
 
+        categorical_features = get_categorical_cols_with_df_with_override(baseline_df,
+                                                                              args.override_numerical_features,
+                                                                              args.override_categorical_features)
         baseline_df = baseline_df.toPandas()
-        task_type = determine_task_type(args.task_type, args.target_column, baseline_df)
+        # lightgbm does not support features that can't be converted to non-bool, int, float features
```

Comment:
what's non-bool type?
In general I don't quite understand this comment. Do you mean lightgbm only supports features that can be converted to bool, int, float? So we have to filter out more categorical features returned from common logic?

Reply:
I modified the comment. From my understanding:
    # Lightgbm can only support features that can be converted to bool, int, float.
    # If these features can't be converted, we have to mark them as "category" types so lightgbm will ignore them.
    # In our design, we mark all known categorical features (including bool) as "category", 
    # only filter out datetime and timedelta because they can be converted to int

I will ask Hannah to review the feature importance part

Reply:
Now changed to all categorical features will be sent to lightgbm, without filtering out.